src/Keboola/S3Extractor/S3AsyncDownloader.php

```diff
@@ -102,6 +102,9 @@ private function makeCommandPool(): CommandPool
                 $this->filesParameter[$index] = $parameters;
             },
             'fulfilled' => function (ResultInterface $result, int $index) {
+                if (($index % 10000) === 0) {
+                    gc_collect_cycles();
+                }
```

Comment:
Toto odporuca AWS, vola sa to po kazdom `10 000`-com subore, kedze inak to bolo pomale a vytazovalo velmi CPU.